src/lib/checkbox/checkbox.ts

```diff
@@ -217,10 +211,6 @@ export class MatCheckbox extends _MatCheckboxMixinBase implements ControlValueAc
     });
   }
 
-  ngAfterViewChecked() {
```

Comment:
We can't actually remove `ngAfterViewChecked()` until a major revision since it removes it from the API.  It will need to be an empty method, but I think the method signature needs to remain.

Reply:
Oki, added it back.